code/src/components/moviedetails.css

```diff
@@ -0,0 +1,122 @@
+.wrapper-details{
+    overflow: hidden;
+}
+
+
+.background-image{
+    min-height: 100vh;
+    width: 100vw;
+    display: flex;
+    background-size: cover;
+    flex-direction: column;
+    justify-content: flex-end;   
+}
+
+
+.movie-poster{
+    display: flex;
+    align-items: flex-end;
+    padding: 50px;
+}
+
+.poster-image{
+    border:solid white 5px;
+}
+
+.summary{
+    display:flex;
+    flex-direction: column;
+    margin:20px;
+}
+
+.movie-name{
+    color:white;
+    margin-bottom:1em;
+}
+
+.info-details{
+    color:white;
+    max-width: 400px;
+    margin:0;
+}
+
+.votes{
+    font-size: 18px;
+    color: red;
+    font-weight: 400;
+    margin-left: 10px;
+}
+
+
+.backbutton{
+    background-color: transparent;
+    background-repeat: no-repeat;
+    border: none;
+    cursor: pointer;
+    overflow: hidden;
+    outline: none;
+    color:white;
+    font-size: 20px;
+    display:flex;
+    padding-left: 3em;
+    padding-top: 2em;
+    position: absolute;
+
+}
+
+.button-text{
+    font-size: 16px;
+    font-weight: bold;
+    margin:0;
+    margin-left: 0.5em;
+}
+
+AiOutlineArrowLeft {
+    font-size: 24px;
+}
+
+@media (min-width: 768px) and (max-width: 1024px) {
+    .movie-poster{
+    padding:0;
+    }
+
+    .movie-name{
+        font-size: 32px;
+        margin-bottom: 0.5em;
+    }
+
+    .votes{
+        font-size: 24px;
+        margin-left: 15px;
+    }
+
+    .info-details{
+        font-size: 18px;
+    }
+
+    .backbutton{
```

Comment:
the Details button ‘Movies’ is not at the same position as in mobile and desktop view, is that done by purpose?